Dockerfile.cypress

```diff
@@ -17,5 +17,6 @@ RUN npm install
 RUN sh download-clis.sh
 
 RUN ["chmod", "+x", "./start-cypress-tests.sh"]
+RUN ["chmod", "+x", "./run-prow-e2e.sh"]
```

Comment:
This is not needed because you have committed the new script with executable bit enabled:
```
-rwxr-xr-x    1 kcormier  staff    1392 Jun  7 17:14 run-prow-e2e.sh
-rw-r--r--    1 kcormier  staff    2696 May 27 17:18 start-cypress-tests.sh
```
If you set the executable bit for `start-cypress-tests.sh` in Git, you could remove the line above as well.

Reply:
(That's the way I prefer to do it so that the scripts are ready for anyone to execute as soon as they check them out.)

Reply:
Ah nice! Will refactor, btw do we need travis related variables? 
- TRAVIS_BUILD_WEB_URL
- TRAVIS_REPO_SLUG
- TRAVIS_PULL_REQUEST